src/coreclr/jit/morph.cpp

```diff
@@ -4446,6 +4722,7 @@ GenTree* Compiler::fgMorphMultiregStructArg(GenTree* arg, fgArgTabEntry* fgEntry
 #endif // DEBUG
 
 #ifndef UNIX_AMD64_ABI
+#if !defined(TARGET_LOONGARCH64)
```

Comment:
remove: this is unnecessary since lvIsHfa() will be false.

Reply:
Thanks